docs/toc.yml

```diff
@@ -9,5 +9,7 @@
   homepage: community/online-resources.md
 - name: Project Chat
   href: https://gitter.im/akkadotnet/akka.net
+- name: Discussion Forum
+  href: https://github.com/akkadotnet/akka.net/discussions
```

Comment:
Figured we could just use this for all repos in the site. Would love to get some more users actively engaged in here.